README.md

```diff
@@ -41,7 +41,83 @@ The Subnet EVM is compatible with almost all Ethereum tooling, including [Remix,
 - Removed Atomic Txs and Shared Memory
 - Removed Multicoin Contract and State
 
+## Setting the Genesis Allocation
+
+When creating an instance of the subnet-evm, you will need to define the genesis state of the new chain. Part of this includes defining the genesis allocation. The genesis allocation allows you to define starting balances for whatever addresses you want.
+
+Simply define the `alloc` field in the genesis JSON as follows:
+
+```json
+  "alloc": {
+    "8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC": {
+      "balance": "0x52B7D2DCC80CD2E4000000"
+    },
+    "Ab5801a7D398351b8bE11C439e05C5B3259aeC9B": {
+      "balance": "0xa796504b1cb5a7c0000"
+    }
+  },
+```
+
+The keys in the allocation are the hex addresses to provide initial funds with the normal `0x` prefix stripped from the string. The balances are defined as hex strings with the `0x` prefix included and denominated in Wei.
```

Comment:
Can we add a reference for hexadecimal https://en.wikipedia.org/wiki/Hexadecimal,

> defined as a hexadecimal number in string with the `0x` prefix.

Also, explain 1 AVAX in C/subnet-evm chain is denominated by 10^18 units of nano avax?

Reply:
I found this link helpful https://www.rapidtables.com/convert/number/hex-to-decimal.html but not sure if we want to include that, or there's any better link/tool

Reply:
Good comments, will add some small tweaks.